okio/src/commonMain/kotlin/okio/-Platform.kt

```diff
@@ -16,6 +16,14 @@
 
 package okio
 
+@ExperimentalFilesystem
```

Comment:
Any value to having experimental annotations on internal members?

Reply:
Without it the code won't compile!